src/extension.ts

```diff
@@ -110,7 +110,14 @@ const createHelperExecutable = (
 
 const authenticateAdo = async (context: vscode.ExtensionContext) => {
   try {
-    await getAccessToken();
+    var scopes = ["499b84ac-1321-427f-aa17-267ca6975798/.default"];
+
+    var tenantID = vscode.workspace.getConfiguration("adoCodespacesAuth").get('tenantID');
```

Comment:
`var` is now considered obsolete in most modern JS codesbases. Prefer `let` or `const`.